autogen/agentchat/conversable_agent.py

```diff
@@ -210,6 +353,77 @@ def reply_func(
             },
         )
 
+    def _register_for_reply(
```

Comment:
Just to avoid duplicate docs. Maybe we can have a note that refers to public function in the Register class

Reply:
Right. I think we should eventually make those methods private as well to avoid confusion.

Reply:
done